tools/foreignkeyGenerator/changes.php

```diff
@@ -122,12 +122,6 @@
             '#unsigned' => true,
         ),
     ),
-    'tax_rules_group' => array(
```

Comment:
Is this related to PHPStan 🤔  ? If yes can you explain how this is marked as false by a static analysis

Reply:
This key is a duplicated to the line 27.